libs/ssl-config/build.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+archivesBaseName = 'elasticsearch-ssl-config'
```

Comment:
It would be preferable to alter `project.name` in `settings.gradle` as that would have effects across the board, including what you are looking for here. Also means that if we were to publish this the  correct pom would be generated without additional configuration and better visibility in the IDE. 

There are some additional changes needed in `settings.gradle` to keep Eclipse happy, see the discussion in #36477 for a similar change.

Reply:
@atorok I pushed a5d0144f which I think covers what you've recommended.
Can you let me know if there's more we need to do?